### ISSUE TITLE
Consumer lag as time.

### ISSUE THREAD

**Issue by wy100101** [danielqsj/kafka_exporter#16]
The current consumer lag is just a count of missed messages, but sometimes it is useful to know how far back in time a consumer group is. If I look at a consumer that is 10,000 messages behind, I can't easily tell if the client is 1 minute/hour/day behind.

I'm not sure how hard it would be to export the timestamp of the latest offset and the consumer's current offset, but it would be really useful.

**Comment by maxtar** [comment]
+1

**Comment by danielqsj** [comment]
@wy100101 @maxtar Kafka exporter is stateless now. Pls using Grafana to show lags curve. It's easy to find out the timestamp.

**Comment by maxtar** [comment]
@danielqsj i don't understand. How I can find time last commited and last message in topic using grafana?
Please write example.

**Comment by danielqsj** [comment]
@maxtar When using grafana, you could find out when the lags starts getting higher and the consumer offset remains the same.

**Comment by maxtar** [comment]
@danielqsj No. I mean I no need when it happend. I want to see time from message last committed and time from last offset message. I think I need consume message from all topic partitions, not from kafka_topic?

**Comment by wy100101** [comment]
Let me see if I can explain this better. Right now, the exporter I can see how many messages behind a consumer is, but I don't know how far back in time they are.  being 1000 messages behind could mean I 10 minutes behind, or it could mean I'm 10 hours behind.

Since all messages also have a timestamp associated with them, I was hoping we could get that timestamp exported along with the offset for the latest offset and each consumer group. I did a quick poke around the sarama libs, but I didn't see an easy way to get that information.
